bump_version.sh

```diff
@@ -26,7 +28,10 @@ else
       # as a result of macOS using the BSD version of sed
       tmp_file=/tmp/version.$$
       sed "s/$old_version_regex/$new_version/" $VERSION_FILE > $tmp_file
-      mv $tmp_file $VERSION_FILE
```

Comment:
I don't think this line should have been removed.

Reply:
Extreme apologies,  restored in: https://github.com/cisagov/awssh/pull/27/commits/5ef42bcca3ca740af0649ab401eed5233372342f